backend/src/main/java/de/neuefische/backend/model/RegistrationData.java

```diff
@@ -0,0 +1,15 @@
+package de.neuefische.backend.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class RegistrationData {
+    String name;
+    String password;
+    String passwordVerify;
```

Comment:
If you check if both passwords match in the frontend you can remove passwordVerify

Reply:
It seems to be the easier solution. 👍